app/views/_includes/applications/fs1.njk

```diff
@@ -0,0 +1,56 @@
+<h2 class="govuk-heading-m govuk-!-font-size-27 govuk-!-margin-top-7 govuk-!-margin-bottom-2 ">Personal details</h2>
+
+{{ govukSummaryList({
+  rows: [{
+    key: {
+      text: "First name"
+    },
+    value: {
+      text: "Patrick"
+    }
+  }, {
+    key: {
+      text: "Last name"
+    },
+    value: {
+      text: "O’Connell"
+    }
+  },  {
+    key: {
+      text: "Date of birth"
+    },
+    value: {
+      text: "19 June 1991"
+    }
+  },{
+    key: {
+      text: "Nationality"
+    },
+    value: {
+      text: "Irish"
+    }
+  },{
+    key: {
+      text: "Have you lived in the UK or Ireland for 3 years or more?"
```

Comment:
We’ve changed this question in the prototype, to reflect that the 3 years which count at the the ones leading up to the course start date (which might be a year after applying).

```suggestion
      text: "Will you have lived in the UK in the 3 years before your teacher training starts?"
```